pkg/providers/cloudstack/template.go

```diff
@@ -66,20 +50,14 @@ func getWorkerNodeGroupMachineSpec(clusterSpec v1alpha1.ClusterSpec, machineConf
 		}
 	}
 
-	if len(workerNodeGroupMachineSpecs) == 0 {
-		return workerNodeGroupMachineSpecs, errors.Errorf("getting worker node group configs")
-	}
```

Comment:
The cluster config is already being validated [here](https://github.com/aws/eks-anywhere/blob/c98621273ade48012679a66f9bcc62e72f40a49d/pkg/api/v1alpha1/cluster.go#L384), so there is no need for this